src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -54,51 +54,104 @@
 				<p v-else-if="$v.email.$error" class="input-error tw-text-danger tw-text-base tw-mb-2">
 					Valid email required.
 				</p>
-				<kv-checkbox
-					data-testid="basket-guest-terms-agreement"
-					id="termsAgreement"
-					name="termsAgreement"
-					class="checkbox tw-text-small tw-mb-2"
-					v-model="termsAgreement"
-					@update:modelValue="$kvTrackEvent(
-						'basket',
-						'click-terms-of-use',
-						'I have read and agree to the Terms of Use and Privacy Policy',
-						$event ? 1 : 0
-					)"
-				>
-					I have read and agree to the
-					<a
-						:href="`https://${this.$appConfig.host}/legal/terms`"
-						target="_blank"
-						title="Open Terms of Use in a new window"
-					>Terms of Use</a>
-					and
-					<a
-						:href="`https://${this.$appConfig.host}/legal/privacy`"
-						target="_blank"
-						:title="`Open Privacy ${enableCommsExperiment ? 'Notice' : 'Policy' } in a new window`"
-					>Privacy {{ enableCommsExperiment ? 'Notice' : 'Policy' }}</a>.
-					<p v-if="$v.termsAgreement.$error" class="input-error tw-text-danger tw-text-base">
-						You must agree to the Kiva Terms of service & Privacy
-						{{ enableCommsExperiment ? 'Notice' : 'Policy' }}.
+				<template v-if="enableRadioBtnExperiment">
+					<fieldset class="tw-flex tw-flex-col tw-gap-2 tw-mt-1 tw-mb-2 tw-text-small">
+						<kv-radio
+							value="1"
+							name="reportComms"
+							v-model="selectedComms"
+							v-kv-track-event="[
+								'basket',
+								'click',
+								'marketing-updates',
+								// eslint-disable-next-line max-len
+								'Receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.',
+							]"
+							:class="{'radio-error': $v.selectedComms.$error}"
+						>
+							<!-- eslint-disable-next-line max-len -->
+							Receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.
+						</kv-radio>
+						<kv-radio
+							value="0"
+							name="reportComms"
+							v-model="selectedComms"
+							:class="{'radio-error': $v.selectedComms.$error}"
+							v-kv-track-event="[
+								'basket',
+								'click',
+								'marketing-updates',
+								// eslint-disable-next-line max-len
+								'Dont receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.',
```

Comment:
Do we need "You can unsubscribe at any time." in this section? It's confusing b/c this is the opt-out so I shouldn't be subscribed and thus shouldn't have to "unsubscribe" at ANY TIME!